app/listeners/app.init/initPassport.js

```diff
@@ -22,7 +22,7 @@ function initPassport (app) {
             .send(userData)
             .end(function (err, resp) {
             if (err) {
-                return done (err,false);
+                return done (null,false);
```

Comment:
So that is a little tricky...if we return the error the app takes you to a new page that just has the error printed on the screen. Since that isn't exactly what we want I switched it to null so that it would at least take us back to the login page. I might have an idea on how to implement some error handling on that. Give me a little bit to see if I can get it to work.